massgov/openmass
# DP-22604: Add drag and drop capabilities to the Mass.gov authoring experience

**Description:**
Added drag and drop capabilities to the Mass.gov authoring experience for curated_lists and info_details content types by enabling the Paragraphs EXPERIMENTAL field widget for the Sections field.


**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22604

**To Test:**
- Login edit an Info Details node.
- Click on the "Content" tab.
- Click on the three dots to the right of the new "Collapse All" button for the "Sections" field.
- Select "Drag & Drop".
- [ ] Verify the interface works as expected, allowing you to move nested paragraphs to other paragraphs and save.  
- Edit a Curated List node.
- Click on the "Content" tab.
- Click on the three dots to the right of the new "Collapse All" button for the "Sections" field.
- Select "Drag & Drop".
- [ ] Verify the interface works as expected, allowing you to move nested paragraphs to other paragraphs and save.
- Edit a Service node.
- Click on the "Content" tab.
- Click on the three dots to the right of the new "Collapse All" button for the "Link Group" field.
- Select "Drag & Drop".
- [ ] Verify the interface works as expected, allowing you to move nested paragraphs to other paragraphs and save.
- Edit a Guide node.
- Click on the "Content" tab.
- Click on the three dots to the right of the new "Collapse All" button for the "Section" field.
- Select "Drag & Drop".
- [ ] Verify the interface works as expected, allowing you to move nested paragraphs to other paragraphs and save.
- Edit a Topic page node.
- Click on the "Content" tab.
- Click on the three dots to the right of the new "Collapse All" button for the "Content card groups" field.
- Select "Drag & Drop".
- [ ] Verify the interface works as expected, allowing you to move nested paragraphs to other paragraphs and save.
- Edit a Binder node.
- Click on the "Content" tab.
- Click on the three dots to the right of the new "Collapse All" button for the "Pages" field.
- Select "Drag & Drop".
- [ ] Verify the interface works as expected, allowing you to move nested paragraphs to other paragraphs and save.

Test other new features that can be disabled: Duplicate, Collapse/Edit all, Add above.

**Screenshots/GIFs:**

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
